### DISCUSSION TITLE
not-found.js doesn't load Tailwind styles in my Route Group

### DISCUSSION THREAD

**Discussion by hadikman** [vercel/next.js#63634] [Help]
### Summary

Hi
Thank you for your help in advance

I have these structure files, a **RootLayout** without the RootPage, and a **RootNotFound**.

**Issue:**
My issue is about loading "global.css" when I go to a path that doesn't exist (404 page).
When not-found.js is loaded, everything is fine except Tailwind styles.

**The app structure:**
![Screenshot 2024-03-23 091659](https://github.com/vercel/next.js/assets/71704773/aec8de54-b167-415d-b342-8ee54d29a5f4)

**Tailwind styles not loaded:**
![Screenshot 2024-03-23 092721](https://github.com/vercel/next.js/assets/71704773/147cb370-afe8-4a24-9a70-8f9e1e93eeee)

**layout.js:**
```
import ReactQueryProvider from '@/react-query-provider'
import { VAZIRMATN_FONT } from 'util/share-font'
import { dehydrate, HydrationBoundary,QueryClient } from '@tanstack/react-query'
import { getProductsAction } from 'action/products/get-products'

import './globals.css'

export const metadata = {
  title: 'Acme',
}

const queryClient = new QueryClient()

await queryClient.prefetchQuery({
  queryKey: ['products'],
  queryFn: () => getProductsAction(),
})

export default async function RootLayout({ children }) {
  return (
    <html className={VAZIRMATN_FONT.className}>
      <body>
        <ReactQueryProvider>
          <HydrationBoundary state={dehydrate(queryClient)}>
            {children}
          </HydrationBoundary>
        </ReactQueryProvider>
      </body>
    </html>
  )
}
```

**not-found.js**
```
export default function RootNotFound() {
  return (
    <main className="grid h-svh max-h-svh w-full place-content-center place-items-center">
      <div>
        <h2 className="inline text-5xl font-bold">404</h2>
        <p className="text-xl font-bold">The page not found!</p>
      </div>
    </main>
  )
}
```
The only way I can see styles is when I delete **.next folder**. For first time everything is fine but when I restart the app, the issue still persist 😑
I don't know, maybe I need a page.js (RootPage) at root, but because I used route groups I think I will have a conflict with the page.js at (rest-routes).

Thanks you 🌹

**Comment by 0-don** [comment]
having the same issue
